### DISCUSSION TITLE
ERROR: Selected branding is not supported by this firmware

### DISCUSSION THREAD

**Discussion by pfichtner** [Freetz-NG/freetz-ng#708] [Allgemein] [answered]
Der Bau älterer Firmwares (z.B. 3270/W920V aka 7570) schlagen fehl: 

```
applying patch file ./patches/scripts/100-aliens.sh
applying patch file ./patches/scripts/101-enforce_branding.sh
  enforcing branding ''
ERROR: Selected branding is not supported by this firmware
make[1]: *** [Makefile:388: firmware-nocompile] Error 1
make: *** [Makefile:46: envira] Terminated
Build step 'Shell ausführen' marked build as failure
Archiviere Artefakte
Finished: FAILURE
```

Der Build findet wöchtentlich statt, die .configs sind zum Teil monatealt und werden vor dem jeweiligen Build mit `make olddefconfig` auf den neusten Stand gebracht. Muss ich die .config "händisch" aktualisieren, etwas hinzufügen/entfernen, habe ich in den configs was drin was bislang nicht störte, jetzt aber ein Problem darstellt oder ist das ein Fehler in Freetz-NG? 

```
❯ grep -i branding */.config
3270/.config:FREETZ_AVM_HAS_BRANDING_avm=y
W920V/.config:FREETZ_AVM_HAS_BRANDING_avme=y
```

**Comment by shtaraen** [comment]
Hi, bei mir kommt ein ähnlicher Fehler. Ich kompiliere auf Freetz-Linux, für eine FritzBox 7240.
Zum Schluss steht bei mir jedoch FINISHED, was leicht irritierend ist - kein Image wurde erstellt. Zu STEP 3 kommt es erst gar nicht. Ich hab das Ganze mehrmals wiederholt mit unterschiedlichen Optionen auch bzgl. Branding, aber immer das gleiche Resultat.

![image](https://user-images.githubusercontent.com/34365875/213939032-72661d4e-645a-4bfa-9403-63612fb06a5e.png)

**Comment by fda77** [comment]
Wie ihr euch evlt vorstellen könnt trat das bei mir nicht auf. Und meine Motivation liegt bei ca 0 alle möglichen Einstellungen im menuconfig/.config durchzuprobieren den Fehler auch zu bekommen

**Reply by shtaraen** [reply]
Meine Config hab ich komplett auf default gelassen, bis auf das Hardware-Modell. Hast du evtl. einen Vorschlag woran es liegen könnte?

**Comment by fda77** [comment, marked as answer]
d906888e6530d496a91d6ad5f272b51150ded9cb
